app/tooling/sales-admin/main.go

```diff
@@ -98,19 +98,6 @@ func processCommands(args conf.Args, log *logger.Logger, cfg config) error {
 			return fmt.Errorf("migrating database: %w", err)
 		}
 
-	case "seed":
```

Comment:
You can't remove this from the admin. You still need the ability to do this for the project.